domain/domain.py

```diff
@@ -196,12 +200,19 @@ def generate_domain(self):
             tid = row['_tid_']
             app = []
             for attr in self.active_attributes:
+                # TODO(richardwu): relax domain prune here: simply take all
+                # values with at least one co-occurrence
                 init_value, dom = self.get_domain_cell(attr, row)
                 init_value_idx = dom.index(init_value)
+                # we will use an Estimator model for weak labelling below, which requires
+                # the full pruned domain first
+                weak_label = init_value
+                weak_label_idx = init_value_idx
+                fixed = CellStatus.not_set.value
                 if len(dom) > 1:
                     cid = self.ds.get_cell_id(tid, attr)
                     app.append({"_tid_": tid, "attribute": attr, "_cid_": cid, "_vid_":vid, "domain": "|||".join(dom),  "domain_size": len(dom),
-                                "init_value": init_value, "init_index": init_value_idx, "fixed":0})
+                                "init_value": init_value, "init_index": init_value_idx, "weak_label": weak_label, "weak_label_idx": weak_label_idx, "fixed": fixed})
```

Comment:
Can we replace `fixed` with `CellStatus.NOT_SET.value` instead of making it a variable that is not changed and not used later

Reply:
Good catch.